i18n/translations/fr/form-builder.json

```diff
@@ -209,17 +209,17 @@
       "text3": "to use this form element."
     },
     "fileInputWithApi": {
-      "description": "A way to add supporting documentation. [FR]",
+      "description": "Un moyen d'ajouter de la documentation à l'appui.",
       "trialFeature": {
-        "title": "Trial feature [FR]",
-        "text1": "This feature has some limitations in the short term as we learn more. In order to use File upload, you must: [FR]",
-        "bullet1": "Set your response delivery to API.",
-        "bullet2": "Files uploaded to an individual form will have a maximum file size of {{BODY_SIZE_LIMIT_WITH_FILES}} MB. [FR]",
-        "bullet3": "All files will be scanned on submission. Please share the form with people you trust. [FR]"
+        "title": "Fonctionnalité à l'essai",
+        "text1": "Certaines limitations s'appliquent. Pour utiliser le téléverseur de fichiers, vous devez :",
+        "bullet1": "Récupérer les réponses au formulaire par API.",
+        "bullet2": "Les fichiers téléversés dans un formulaire peuvent avoir une taille maximale combinée de {{BODY_SIZE_LIMIT_WITH_FILES}} MB.",
+        "bullet3": "Tous les fichiers seront analysés après avoir été soumis afin d'identifier des fichiers malveillants."
       },
       "recommendations": {
         "title": "Recommendations",
-        "text1": "Add fields to gather contact information in case you need to support those with the files they have attached. [FR]"
+        "text1": "Add fields to gather contact information in case you need to support those with the files they have attached. [FR] "
```

Comment:
Missing this one.

<img width="500" alt="Screenshot 2025-06-11 at 7 08 48 AM" src="https://github.com/user-attachments/assets/14354ddb-b52a-457b-92aa-e31fa23443bf" />